sdk/typescript/runtime/main.go

```diff
@@ -218,23 +177,125 @@ func (t *TypescriptSdk) Base(runtime SupportedTSRuntime) (*Container, error) {
 		return dag.Container().
 			From(nodeImageRef).
 			WithoutEntrypoint().
+			WithExec([]string{"corepack", "enable"}).
 			// Install default CA certificates and configure node to use them instead of its compiled in CA bundle.
 			// This enables use of custom CA certificates if configured in the dagger engine.
 			WithExec([]string{"apk", "add", "ca-certificates"}).
 			WithEnvVariable("NODE_OPTIONS", "--use-openssl-ca").
 			WithMountedCache("/root/.npm", dag.CacheVolume(fmt.Sprintf("mod-npm-cache-%s", nodeVersion))).
+			// WithMountedCache("/usr/local/share/.cache", dag.CacheVolume(fmt.Sprintf("mod-yarn-cache-%s", nodeVersion))).
```

Comment:
Why is this commented out? Either it should be enabled or removed.

Also, this doesn't look like the right path.

Reply:
> Add back cache key on yarn module. I still need to understand what's going on with that but we can save it for later, I think it will take some time and current resources on the web are not that helpful in our specific context.

I don't want to remove it, it's a workaround to fix an issue with yarn cache, but yeah I think I miss wrote the cache path too

Reply:
Howerver, I added a comment to explain why this line is commented and fixed the path ([387eb49](https://github.com/dagger/dagger/pull/7096/commits/387eb49a0da4916c410830d8cbc6a72cd086dde2)